.github/workflows/ci.yml

```diff
@@ -41,10 +41,8 @@ jobs:
       matrix:
         lint-command:
           - "bandit -r vies -x tests"
-          - "black --check --diff ."
-          - "flake8 ."
-          - "isort --check-only --diff ."
-          - "pydocstyle ."
+          - "ruff format --check --diff ."
+          - "ruff check ."
```

Comment:
Do you need both? If I am not mistaken, `check` will include a format check too, right?